pkg/parcacol/querier.go

```diff
@@ -102,7 +107,8 @@ func (q *Querier) Labels(
 				}
 
 				val := stringCol.Value(i)
-				seen[strings.TrimPrefix(val, "labels.")] = struct{}{}
+				l := strings.TrimPrefix(strings.TrimPrefix(val, ColumnLabelsPrefix), ColumnPprofLabelsPrefix)
```

Comment:
based on the prefix we can tell what kind of profile it is and return that to the frontend, and only the frontend will unify it into a single list

Reply:
I don't quite understand 😄 

I assumed that since we removed the prefix for `labels.` we should do the same irrespective of where the prefix comes from.

Do you mean we should only trim the `labels.` prefix and the frontend is going to take care of removing the `pprof_labels.` prefix?